src/Rules.Framework/Generics/GenericContentType.cs

```diff
@@ -0,0 +1,29 @@
+namespace Rules.Framework.Generics
+{
+    using System;
+
+    /// <summary>
+    /// Defines generic content type
+    /// </summary>
+    public struct GenericContentType : IEquatable<GenericContentType>
+    {
+        /// <summary>
+        /// Gets or sets the display name.
+        /// </summary>
+        /// <value>The display name.</value>
+        public string DisplayName { get; set; }
```

Comment:
Please review this one according to the endgame of the discussion on `GenericConditionType`.

Reply:
Done